packages/sdk/package.json

```diff
@@ -87,6 +88,7 @@
     "@typescript-eslint/eslint-plugin": "^5.0.0",
     "@typescript-eslint/parser": "^4.13.0",
     "adm-zip": "^0.5.9",
+    "botbuilder-core": ">=4.15.0 <5.0.0",
```

Comment:
why we need to add it to devDependencies?

Reply:
It's only used in test files so I moved it to devDependendies.